FeatureSuggestion/ShaktiPrasadLenka.md

```diff
@@ -0,0 +1,26 @@
+#### Considering MVP, lets have one simple template
+##### 1. Form of the template will have below Sections:
+- Name
```

Comment:
Awesome suggestions. Feedback - You can also mention that these sections are not mandatory.  Use can choose to remove any of the selected section.

Reply:
@ajay109458  yes, Lets have [-] symbol denoting remove section feature.